src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -47,9 +47,10 @@
  * <ul>
  * <li>
  * Property {@code aliasList} - Specify aliases for check names that can be used in code
- * within {@code SuppressWarnings}.
+ * within {@code SuppressWarnings} in a format of comma separated attribute=value entries.
+ * The attribute is the fully qualified name of the Check and value is its alias.
  * Type is {@code java.lang.String[]}.
- * Default value is {@code null}.
+ * Default value is {@code ""}.
```

Comment:
Not expected , but I will improve after merge of blocker PR.

Reply:
How is it not expected and how will it be improved?

Reply:
By doing https://github.com/checkstyle/checkstyle/pull/13964#discussion_r1399822130

Reply:
How will going against the setter improve the default value? There is no default value in the setter.

Reply:
I have idea to:
If field is not found, default is null for sure. We have scalar types and array. 
For scaral type null is null, for array it will be `{}`

Reply:
fixed.